dashboard/config/scripts/allthethings.script

```diff
@@ -244,6 +245,3 @@ level 'allthethings_fish_tank'
 stage 'Dance Lab'
 level 'allthethings HOC Dance 1'
 level 'allthethings HOC Dance 10 Measures'
-
-stage 'Mini Rubric'
-assessment 'allthethings-mini-rubric'
```

Comment:
Intentionally removing the mini-rubric stage?

Reply:
oops, no! fixed.